pkg/mover/chart.go

```diff
@@ -182,33 +187,25 @@ func NewChartMover(req *ChartMoveRequest, opts ...Option) (*ChartMover, error) {
 		}
 	}
 
-	patternsRaw, err := loadPatterns(req.Source.ImageHintsFile, chart, cm.logger)
-	if err != nil {
-		return nil, err
-	}
-
-	if patternsRaw == nil {
+	if cm.rawHints == nil {
```

Comment:
Where is the magic now that sets this value? I preferred the previous way which was explicit.

Reply:
Well the thing is, we load the hints differently if it is from the bundle or the chart. So that makes sense to go to the **if** handling that instead of having to repeat the if in several places.

This is a post check after the loaders, one or the other, so that we ensure hints were loaded or we error otherwise.

Reply:
decoupled hints loading to fix this

Reply:
decoupled hints loading to fix this